spec/models/analytics/patient_set_analytics_spec.rb

```diff
@@ -119,6 +118,7 @@
         Date.new(2019, 3, 03) => 0,
         Date.new(2019, 3, 10) => 0,
         Date.new(2019, 3, 17) => 0,
+        Date.new(2019, 3, 24) => 0,
```

Comment:
These tests have been failing on `master` since last week. I don't think they correctly respecting time right now. Can we nest them under having them frozen in time for the time they span?